cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/ControlDependenceGraphPass.kt

```diff
@@ -60,7 +60,7 @@ open class ControlDependenceGraphPass(ctx: TranslationContext) : EOGStarterPass(
     }
 
     /**
-     * Computes the CDG for the given [functionDeclaration]. It performs the following steps:
+     * Computes the CDG for the given [FunctionDeclaration]. It performs the following steps:
```

Comment:
Should probably be `[startNode]` (and the condition in line 76 will have to be updated to an EOG Starter without prevEOG, etc. in the future)